src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtilsUnitTest.java

```diff
@@ -30,40 +34,223 @@
 
         final ArrayList<Object[]> testArgs = new ArrayList<>();
 
-        final ArrayList<Integer> baseArgs = new ArrayList<>();
-        baseArgs.add(DataSourceUtils.MIN_MAJOR_VERSION_NUMBER);
-        baseArgs.add(DataSourceUtils.MIN_MINOR_VERSION_NUMBER);
-        baseArgs.add(DataSourceUtils.MIN_YEAR_RELEASED);
-        baseArgs.add(DataSourceUtils.MIN_MONTH_RELEASED);
-        baseArgs.add(DataSourceUtils.MIN_DAY_RELEASED);
+        // =============================================================
+        // First do tests that check the major / minor version numbers:
+        // -------------------------------------------------------------
 
-        for ( int offset = -1 ; offset < 2; ++offset ) {
-            for ( int i = 0; i < baseArgs.size(); ++i ) {
+        final int MINOR_IT_MAX_VAL = 20;
```

Comment:
When you check the minor versions, instead of iterating from 0 to 20, you should just check the boundary values 0, MIN_MINOR - 1, MIN_MINOR, MIN_MINOR + 1, MAX_MINOR - 1, MAX_MINOR, MAX_MINOR + 1. After all, you might have a minor version requirement that's > 20 one day.

Reply:
OK - Fixed!